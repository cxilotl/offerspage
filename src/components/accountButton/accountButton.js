import React from 'react';
import PropTypes from 'prop-types';
import cssStyles from './accountButton.module.scss';
import userIcon from "../../user-icon.svg";

const AccountButton = ()=> {
  return(
    <button id="account-btn" name="account-btn" data-testid="exchange-rate" className={ cssStyles.layout } type="button">
      <img src={ userIcon } className={ cssStyles.icon } alt="User icon" />
      <span className={ cssStyles.label }>Jane Smith</span>
    </button>
  );
};

export default AccountButton;