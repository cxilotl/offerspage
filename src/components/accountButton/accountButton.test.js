import React from 'react';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import AccountButton from './accountButton';

describe('AccountButton', () => {

  it('Snapshot', () => {
    const component = renderer.create(
      <AccountButton />
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

});