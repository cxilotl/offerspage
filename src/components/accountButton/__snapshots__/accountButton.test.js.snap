// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccountButton Snapshot 1`] = `
<button
  className="layout"
  data-testid="exchange-rate"
  id="account-btn"
  name="account-btn"
  type="button"
>
  <img
    alt="User icon"
    className="icon"
    src="user-icon.svg"
  />
  <span
    className="label"
  >
    Jane Smith
  </span>
</button>
`;
